app/scripts/directives/nav.js

```diff
@@ -273,10 +291,47 @@ angular.module('openshiftConsole')
         };
 
         $scope.catalogLandingPageEnabled = !Constants.DISABLE_SERVICE_CATALOG_LANDING_PAGE;
-        var select = $elem.find('.selectpicker');
-        var options = [];
 
-        var updateOptions = function() {
+        var contextSelector = $elem.find('.contextselector');
+        var previousContextSelectorValue;
+        $scope.clusterConsoleURL = window.OPENSHIFT_CONSTANTS.TECTONIC_URL;
+        contextSelector
+          .on('loaded.bs.select', function () {
+            setContextPickerSelectedOption(contextSelector);
+          })
+          .on('shown.bs.select', function() {
+            previousContextSelectorValue = $(this).val();
+          })
+          .change(function() {
+            // // unset focus on .dropdown-toggle so outline isn't present
+            // $('.contextselector .dropdown-toggle').blur();
```

Comment:
Remove commented out code? (I assume we want to leave it out for accessibility. Whatever we do, we should try to be consistent with the behavior in Tectonic console.)